.travis.yml

```diff
@@ -12,9 +12,22 @@ services:
 env:
   global:
     - DOCKER_CACHE_FILE=/home/travis/docker/cache.tar.gz
-    - DOCKER_COMPOSE="docker-compose -f compose/travis/docker-compose.yml"
+    - DOCKER_COMPOSE="docker-compose -f docker-compose.test.yml"
+    - DOCKER_COMPOSE_VERSION=1.22.0
+    - UID=$TRAVIS_UID
 
 before_install:
+  # TODO: Remove when travis has a adequate version preinstalled
+  # upgrade to latest docker version to avoid "Forbidden path outside the build context: ../Dockerfile" error
+  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+  - sudo apt-get update
+  - sudo apt-get -y install docker-ce
+  # upgrade to latest docker-compose
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
```

Comment:
This might be better placed in a `install_latest_docker.sh` script to not cludder the `.travis.yml`.

Reply:
> lukasjuhrich marked this conversation as resolved.

?

Reply:
I retroactively thought `comment` would have been better than `request changes`, but you can't change that. So I just marked it as resolved so it wouldn't be a blocker. Which also didn't work.

Reply:
It was a reasonable suggestion, and the fix was easy enough, so I guess requesting changes was fine.